components/size_aware_image.jsx

```diff
@@ -121,22 +126,16 @@ export default class SizeAwareImage extends React.PureComponent {
     render() {
         const {
             dimensions,
+            src,
             ...props
         } = this.props;
 
         Reflect.deleteProperty(props, 'showLoader');
         Reflect.deleteProperty(props, 'onImageLoaded');
         Reflect.deleteProperty(props, 'onImageLoadFail');
 
-        let src;
-        if (!this.state.loaded && dimensions) {
-            // Generate a blank image as a placeholder because that will scale down to fit the available space
-            // while maintaining the correct aspect ratio
-            src = createPlaceholderImage(dimensions.width, dimensions.height);
-        } else if (this.state.error) {
+        if (this.state.error) {
```

Comment:
Checking this before making the placeholder will cause the page to jump if the image fails to load. Are you intending that? Doing this will help solve some issues we were seeing on another PR though, so it might be worth making this change.

Reply:
Changed this to have placeholder even when image fails